src/handbook/development/ops/maintenance.md

```diff
@@ -0,0 +1,81 @@
+---
+navTitle: Infrastructure Maintenance
+---
+
+Periodically, FlowFuse performs maintenance on FlowFuse Cloud resources. This page describes how we manage and execute maintenance tasks around the infrastructure.
+
+# Maintenance scope
+
+The maintenance most often involves updates to the following resources:
+* underlying cluster hosts
+* Kubernetes versions
+* databases
+* networking components
+
+ 
+# Maintenance process
+
+## Planning
+
+The maintenance process is initiated by the Operations Team.
+
+A new [Infrastructure task](https://github.com/FlowFuse/CloudProject/issues/new?template=infra-task.yml) is created to co-ordinate all activities regarding the task - including the technical tasks and communications with customers.
```

Comment:
@ppawlowski @knolleary Can the linked template require the updating of customers to be coordinated?